fdbclient/SystemData.cpp

```diff
@@ -898,6 +898,7 @@ const KeyRef consistencyScanInfoKey = "\xff/consistencyScanInfo"_sr;
 
 const KeyRef encryptionAtRestModeConfKey("\xff/conf/encryption_at_rest_mode"_sr);
 const KeyRef tenantModeConfKey("\xff/conf/tenant_mode"_sr);
+const KeyRef tenantIdPrefixKey("\xff/tenant_id_prefix"_sr);
```

Comment:
The tenant metadata is all defined in `Tenant.h`, so I think we should move this there. We may not want to put it in `TenantMetadataSpecification`, though, because on the management cluster the natural way to use that would write it into the regular key-space.